### PR TITLE
doc: Add release note for the deprecated totalFee option of bumpfee

### PR DESCRIPTION
**Pull request by promag**
Adds release notes for #15996.

### CODE REVIEW

**Comment by ezegom** [comment]
Would it make sense to add the support for feeRate to the release notes as well? 
https://github.com/bitcoin/bitcoin/pull/16492

**Comment by promag** [comment]
You can add a release note for the new option in your PR for now.

**Comment by laanwj** [comment]
ACK 61e7f120bafe9aba3f0db76e72afa9902bc548c7

File: doc/release-notes-15996.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Deprecated or removed RPCs
+--------------------------
+
+- The `totalFee` option of the `bumpfee` RPC has been deprecated and will be
+  removed in 0.20.  To continue using this option start with
+  `-deprecatedrpc=totalFee`.  See the `bumpfee` RPC help text for more details.
```
Comment (MarcoFalke) [inline doc/release-notes-15996.md]: This is not going to be backported, so might as well inline it into the main release notes right now.

Comment (promag) [reply doc/release-notes-15996.md]: Done.

**Pull request merged**